cmd/process-agent/info.go

```diff
@@ -377,7 +367,8 @@ func Info(w io.Writer, _ *config.AgentConfig, expvarURL string) error {
 	var info StatusInfo
 	info.LogFile = ddconfig.Datadog.GetString("process_config.log_file")
 	if err := json.NewDecoder(resp.Body).Decode(&info); err != nil {
-		program, banner := getProgramBanner(Version)
+		clientVersion, _ := version.Agent()
```

Comment:
How will this be different from `agentVersion`?

Reply:
Good question! Since we are dealing with two different agents here, they can each have their own separate versions. `clientVersion` is the version of the agent that is requesting the `info` payload whereas `agentVersion` is the version of the agent sending the `info` payload.